inifix/format.py

```diff
@@ -2,44 +2,86 @@
 import os
 import re
 import sys
+import warnings
 from typing import Optional
 from typing import Sequence
 
 from inifix.io import load
 
 NAME_COL_MINSIZE = 10
 COMMENT_COL = 80
+DEFAULT_PADDING_SIZE = 4
+
+
+def ljust(string: str, width: int) -> str:
```

Comment:
`_` is to signal that I don't want this function to be considered public API
Also, many other functions/methods in inifix use `s` as a first argument when it's a str (it's ugly, but consistency prevails, and it won't matter in the future when these arguments become positional-only)
```suggestion
def _ljust(s: str, width: int) -> str:
```